.github/workflows/test.yml

```diff
@@ -22,9 +22,14 @@ jobs:
      - name: Build E300 on Arty
        uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
        with:
-         command: -x 'makeRTL e300ArtyDUTPlan'
+         command: -x 'makeRTL e300ArtyDUTPlan | getDUTVsrcs | findFailFn getPathResult'
 
      - name: Build U500 on VC707
        uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
        with:
-         command: -x 'makeRTL u500VC707DUTPlan'
+         command: -x 'makeRTL u500VC707DUTPlan | getDUTVsrcs | findFailFn getPathResult'
+
+     - name: Build U500 on VCU118
+       uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
+       with:
+         command: -x 'makeRTL u500VCU118DUTPlan | getDUTVsrcs | findFailFn getPathResult'
```

Comment:
You may want to consider splitting these into three separate jobs so that they can run in parallel, depending on how slow the builds are.

Reply:
lol - your comment beat my push!

Reply:
I'll note that by splitting them apart, the scala compilation now happens three times, though.

Reply:
Yeah, but I think the engineering time saved by reducing the latency of a full build is more than worth whatever compute time we are wasting building the same thing three times. We might not even be paying anything since this is an open source repo, anyway.